Application/RSBot/Views/ScriptRecorder.cs

```diff
@@ -200,11 +217,66 @@ private void OnRequestTeleport(uint destination, string npcCodeName)
         /// </summary>
         private void OnTerminateVehicle()
         {
-            if (!_recording) return;
+            if (!_recording) 
+                return;
 
             txtScript.AppendText("dismount\n");
         }
 
+        /// <summary>
```

Comment:
What's the purpose of all those event listeners? They're doing nothing atm.

Reply:
With next update, i will use them